app/views/moments/show.html.erb

```diff
@@ -43,6 +43,16 @@
   </div>
 <% end %>
 
+<div class="smallMarginTop">
+  <div class="label"><%= label_tag "What resources could help?" %> </div>
+  <ul>
+    <% @resources.take(3).each do |item|  %>
+      <li><%= link_to item['name'], item['link'] %></li>
+    <%end %>
+    <li><%= link_to  'More...', resources_path %></li>
```

Comment:
For the more link, I think we should also have some tag(s) pre-selected e.g. `https://www.if-me.org/resources?filter[]=hotlines` based on the preprocessing you've done.